ystockquote.py

```diff
@@ -21,8 +21,9 @@
     from urllib.parse import urlencode
 except ImportError:
     # py2
-    from urllib2 import Request, urlopen
     from urllib import urlencode
+    from urllib2 import Request, urlopen
+    import datetime
```

Comment:
i believe it should be alphabetical by module name.. so:

import datetime
from urllib import urlencode
from urllib2 import Request, urlopen